src/wireviz/wv_helper.py

```diff
@@ -53,6 +56,41 @@ def nested_html_table(rows):
     html.append('</table>')
     return html
 
+def html_colorbar(color):
+    return f'<tdX bgcolor="{wv_colors.translate_color(color, "HEX")}" width="4">' if color else None
+
+def html_image(node):
+    if not node.image:
+        return None
+    size = html_size_attr(node.image_size)
+    # The leading attributes belong to the preceeding tag. See where used below.
+    html = f'{"".join(size)}><img scale="{node.image_scale}" src="{node.image}"/>'
```

Comment:
why do `"".join(size)` here? why not make `html_size_attr()` directly return a string that can be included as-is?

Reply:
Because I also need to test if it has a `fixedsize` attribute set, and I then I would need to test if the resulting string ends with `fixedsize="TRUE"` by repeating this string is two different functions or defining it as a global string - both resulting in less readable code, IMHO. A third option is to repeat the test of the input list, but that is repeating the same code twice.

Reply:
A fourth option is to always include the extra nested table and then not needing the test if it has a `fixedsize` attribute set in this function.

Reply:
Why do you need to check the output of `html_size_attr()` to know if it has the `fixedsize` attribute, instead of getting it from the image attributes [as I suggest](https://github.com/formatc1702/WireViz/pull/153/files/9a6522395913788c6583aeee96342203fe007cbe#r470489101)

Reply:
> A fourth option is to always include the extra nested table and then not needing the test if it has a `fixedsize` attribute set in this function.

This is also an option, since the extra table doesn't really hurt anybody :-)

Reply:
A new `image` class might change this picture completely.

Reply:
Just to make sure: Are you planning on implementing a true `image` class? Or just have an `image` attribute of `dict` type, with `src`, `caption`, `width` etc. as keys? That should probably be enough, right?

Reply:
I prefer a true Image dataclass to obtain the easier syntax `image.src` instead of `image['src']`. The only processing needed are two extra lines in `__post_init__()` of the two classes containing an image.